statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -2894,6 +2894,76 @@ def zvalues(self):
         """
         return self.params / self.bse
 
+    def get_rsquared(self, baseline="rwdrift", **kwargs):
+        # if self.model.k_endog > 1:
+        #     raise NotImplementedError(
+        #         "To be Implemented for Multivariate Model")
+        def sse_finite():
+            d = np.maximum(self.loglikelihood_burn, self.nobs_diffuse)
+            srss = np.sum(self.standardized_forecasts_error[0, d:]**2)
+            f_T = self.forecasts_error_cov[0, 0, -1]
+            return f_T * srss
+        sse = sse_finite()
+        if baseline == "rwdrift":
+            ssdm = np.var(np.diff(self.endog)) * (len(self.endog) - 1)
+            return 1. - sse / ssdm
+        elif baseline == "mean":
+            ssm = np.var(self.endog) * len(self.endog)
+            return 1. - sse / ssm
+        elif baseline == "seasonal":
+            import statsmodels.api as sm
+            seasonal = 12 if 'seasonal' not in kwargs else kwargs['seasonal']
```

Comment:
Instead of a default, I suggest raising an error if `baseline == 'seasonal'` but they didn't provide the `seasonal` argument. 

In the future, we can expand this to take account of the model's seasonal term or the frequency of a Pandas index, if it's given.